right-click-context.el

```diff
@@ -207,15 +207,26 @@ You probably want to just add follows code to your .emacs file (init.el).
     (call-interactively #'mouse-set-point))
   (right-click-context-menu))
 
+;;;###autoload
+(defun right-click-context-click-menu-popup ()
```

Comment:
This command is probably not a command for the end user to execute explicitly.  Can you change the name of this command to `right-click-context--click-menu-popup` as private?

Reply:
Definitely! Renamed!